rosboard/handlers.py

```diff
@@ -201,17 +211,24 @@ def on_message(self, message):
 
             op_name = argv[1].get("op_name")
             args = argv[1].get("args")
+            print("Sending bot to " + str(args))
```

Comment:
Should we keep this print statement?  may want to comment it out

Reply:
Thanks, removed it. Was just used for debugging